client/src/components/ratingsReviews/ReviewsHelperFunc.jsx

```diff
@@ -27,11 +27,11 @@ const helpers = {
         return false;
     },
     generateHelpfulness: (helpNum, id) => {
-        console.log('id', id);
+
         if (helpNum > 0) {
             return (<div id="helpful">
-                <span><p id="helpful-question">Helpful? </p>
-                <p id= {"helpful-link" + id} onClick = {
```

Comment:
I think there are 2 extra spaces on each of these lines. Look into formatting this